pages/vi/vi-planet-installation-and-configuration.md

```diff
@@ -114,6 +114,9 @@ Follow the official [Install Docker Compose](https://docs.docker.com/compose/ins
   - `docker pull treehouses/planet:latest`
   - `docker pull treehouses/planet:db-init`
 
+  - `docker tag treehouses/planet:latest treehouses/planet:local`
```

Comment:
looks good but there is difference between spacing of previous and current tag so could you try to make it same